instance/models/instance.py

```diff
@@ -410,6 +410,27 @@ def ansible_swift_settings(self):
                                 'region': self.swift_openstack_region})
 
     @property
+    def lms_preview_sub_domain(self):
+        """
+        LMS preview sub-domain name (eg. 'preview.master')
+        """
+        return 'preview.{}'.format(self.sub_domain)
```

Comment:
I thought that we had agreed to use `preview-{}` instead of `preview.{}`? And I was hoping this would change the Studio URLs to `studio-{}` at the same time, though that's not a requirement.

See comment 3 and 4 on OC-1594.